app/models/form_profile.rb

```diff
@@ -205,55 +201,30 @@ def prefill
     { form_data:, metadata: }
   end
 
+  def initialize_military_information
+    return {} unless user.authorize :va_profile, :access?
+
+    FormMilitaryInformation.new(initialize_va_profile_prefill_military_information)
+  end
+
   private
 
-  def initialize_military_information_vaprofile
-    military_information_data = {}
-    military_information = HCA::MilitaryInformation.new(user)
+  def initialize_va_profile_prefill_military_information
+    military_information = VAProfile::Prefill::MilitaryInformation.new(user)
 
-    HCA::MilitaryInformation::PREFILL_METHODS.each do |attr|
-      military_information_data[attr] = military_information.public_send(attr)
+    VAProfile::Prefill::MilitaryInformation::PREFILL_METHODS.index_with do |attr|
```

Comment:
why did we change this to `index_with` if we aren't using the index? Could just keep it `.each`, right?

Reply:
Changing it back to `.each` makes about half of the tests the the form_profile_spec fail. In the original block we assign a value to `military_information.public_send(attr)` whereas in the modified code, we don't, and `.each` doesn't inherently return the result of the method calls, while `index_with` does. I'll leave it as `index_with`.

For more context, this is what most of the failures were. That array is the PREFILL_METHODS:
```
      NoMethodError:
        Expected ["compensable_va_service_connected", "currently_active_duty", "currently_active_duty_hash", "discharge_type", "guard_reserve_service_history", "hca_last_service_branch", "is_va_service_connected", "last_discharge_date", "last_entry_date", "last_service_branch", "latest_guard_reserve_service_period", "post_nov111998_combat", "service_branches", "service_episodes_by_date", "service_periods", "sw_asia_combat", "tours_of_duty", "va_compensation_type"] to respond to #to_hash
```

Reply:
ah ok, makes sense